rust/tvm-rt/src/ndarray.rs

```diff
@@ -374,28 +430,27 @@ mod tests {
 
     #[test]
     fn basics() {
-        let shape = &mut [1, 2, 3];
+        let shape = &[1, 2, 3];
         let ctx = Context::cpu(0);
+        println!("before empty");
```

Comment:
Stale helpers, not needed anymore?

Reply:
I'm fine with println! in tests, since the output is hidden by default, but that's just me.

Reply:
Yep! They don't bug me, just flagged them in case they were leftovers.